### ISSUE TITLE
`--oom-avoid-byes` flag doesn't exist in stress-ng used by the debian version of checkbox on ubuntu versions older than 24.04

### ISSUE THREAD

**Issue by tomli380576** [canonical/checkbox#1811]
### Bug Description

The `--oom-avoid-bytes` flag added in this PR #1722 doesn't exist in older, debian version of stress-ng used by checkbox. If we run the `client-cert-desktop-22-04-stress` test plan, this flag will cause the memory and cpu stress tests to immediately terminate.

### Cert-blocker Test Case

- [x] cert-blocker

### To Reproduce

1. Install the debian version of checkbox on a 22.04 or older device
2. `stress-ng --oom-avoid-bytes 10%`

### Expected Result

CPU/Memory stress test jobs run normally

### Actual Result

Job failed because the flag doesn't exist

### Environment

- OS: 22.04
- Type: Desktop
- Checkbox version: 4.4.0-dev54

### Relevant log output

```shell

```

**Comment by pieqq** [comment]
stress-ng is available via [a PPA](https://launchpad.net/~colin-king/+archive/ubuntu/stress-ng). The version there for Jammy is currently 0.18.11 which includes the `--oom-avoid-byes` flag.

When using the Debian version of Checkbox, you should make sure you pull stress-ng from the PPA so that you can run the stressors with the latest version.

**Comment by bladernr** [comment]
Ummm, stress-ng is already being built daily in the cert PPAs (stable and edge, but not beta it seems).  Presumably if you are installing the checkbox debs from either edge or stable, they should pull in stress-ng from edge or stable as well, which are version 0.18.11 as of right now:

```
$ apt-cache policy stress-ng
stress-ng:
  Installed: 0.17.06-1build1
  Candidate: 0.18.11-0~202503191237~ubuntu24.04.1
  Version table:
     0.18.11-0~202503191237~ubuntu24.04.1 500
        500 https://ppa.launchpadcontent.net/checkbox-dev/edge/ubuntu noble/main amd64 Packages
     0.18.11-0~202503191237~ubuntu24.04.1 500
        500 https://ppa.launchpadcontent.net/checkbox-dev/stable/ubuntu noble/main amd64 Packages
 *** 0.17.06-1build1 500
        500 http://archive.ubuntu.com/ubuntu noble/universe amd64 Packages
        100 /var/lib/dpkg/status
```
